docs/pages/guides/gha_basic.md

```diff
@@ -276,6 +285,8 @@ branches that you can use.
   Publish Python packages to PyPI. Supports trusted publisher deployment.
 - [re-actors/alls-green](https://github.com/re-actors/alls-green): Tooling to
   check to see if all jobs passed (supports allowed failures, too).
+- [sigstore/gh-action-sigstore-python](https://github.com/sigstore/gh-action-sigstore-python):
```

Comment:
@henryiii FYI the sigstore action does not have a `release/v1` branch. This is the pattern that I tend to follow, but that action isn't mine and William does not use it.

Reply:
Oops, I missed the heading. I can move it up. Thanks!